test_bifrost.sh

```diff
@@ -3,9 +3,11 @@
 export BIFROST_DB_KEY="${BIFROST_DB_KEY:-$1}"
 export BIFROST_RUN_DIR="${BIFROST_RUN_DIR:-$PWD}"; \
 export BIFROST_CONFIG_DIR="${BIFROST_CONFIG_DIR:-$PWD}"; \
+export BIFROST_DIR=$BIFROST_RUN_DIR; \
```

Comment:
Realized after I made the change that BIFROST_DIR should be wholy contained in pre.sh, per_sample.sh, and post.sh so shouldn't  be needed